rudder-core/src/main/scala/com/normation/rudder/services/system/SupportInfoService.scala

```diff
@@ -66,28 +72,32 @@ class SupportInfoServiceImpl extends SupportInfoService with Loggable {
     try {
       Full(Await.result(RunNuCommand.run(cmd, timeOut), timeOut))
     } catch {
-      case e => Failure(s"An Error has occured when launching support-info script: ${e.getMessage}")
+      case e: Throwable => Failure(s"An Error has occured when launching support-info script: ${e.getMessage}")
     }
   }
 
   // The support script generates an archive in the /tmp folder.
   // We want to get its binary representation into an Array of byte
   // In order for the API to build an InMemoryResponse
 
-  private[this] def getScriptResult() : Box[Array[Byte]] = {
+  private[this] def getScriptResult() : Box[SupportInfoScriptResult] = {
+    import java.net.InetAddress.getLocalHost
 
-    val resultPath = "/tmp/support-info-server.tar.gz"
     Try {
+      val serverHostname = getLocalHost.getHostName
```

Comment:
getLocalHost may fail - you should catch the UnknownHostException and set a default name in this case
also, the security manager need to allow this operation

Reply:
This is in a Try block to prevent exceptions from this function

Reply:
The script already define the name of the file, we just try to find back the file in /tmp, if java could not find back localhost I prefer to have an error, than trying to guess the file.

I think the security manager is only set for javascript engine, not for Rudder